tests/test_consumer/test_handlers/test_runs_handler_airflow.py

```diff
@@ -97,7 +98,10 @@ async def test_runs_handler_airflow(
     assert task_run.status == RunStatus.SUCCEEDED
     assert task_run.started_at == datetime(2024, 7, 5, 9, 4, 20, 783845, tzinfo=timezone.utc)
     assert task_run.ended_at == datetime(2024, 7, 5, 9, 7, 37, 858423, tzinfo=timezone.utc)
-    assert task_run.started_by_user_id is None
+    user_query = select(User).where(User.name == "myuser")
+    user_scalars = await async_session.scalars(user_query)
+    user = user_scalars.one_or_none()
```

Comment:
Could you move this query above the assert? Also dag_run still has no started_by_user_id, is this intentional?